src/main/steps/applicant1/hub-page/right-column/content.ts

```diff
@@ -44,6 +44,8 @@ const en = ({ isDivorce, isApplicant2, userCase }: CommonContent) => ({
     (isApplicant2 ? (userCase?.applicationType === ApplicationType.SOLE_APPLICATION ? RESPONDENT : APPLICANT_2) : '') +
     CHECK_CONTACT_DETAILS
   }">Review your contact details</a>`,
+  certificateOfEntitlementDownloadLink:
+    '<a class="govuk-link" href="/downloads/certificate-of-entitlement" download="Certificate-of-entitlement">View your ‘certificate of entitlement’ (PDF)</a>',
```

Comment:
Can remove this because it's set above already

Could you double check if it should be 
_View your ‘certificate of entitlement’ (PDF)_ or _View the certificate of entitlement (PDF)_
please

Reply:
tbh i thought i did, i checked when i merged master but i guess i missed it. will check thanks